base/loading.jl

```diff
@@ -769,16 +769,14 @@ precompilableerror(@nospecialize ex) = false
 
 # Call __precompile__(false) at the top of a tile prevent it from being precompiled (false)
 """
-    __precompile__(false)
+    __precompile__(isprecompilable::Bool)
 
-Specify that the file calling this function is not precompilable.
+Specify that the file calling this function is not precompilable, defaulting to `true`.
```

Comment:
The wording makes it sound like precompilation is _disabled_ by default. I'd reword as

> Specify whether the file calling this function is precompilable, defaulting to `true`. If this function is not called explicitly, precompilation will be enabled by default.